### DISCUSSION TITLE
Unable to delete team because verify field is not working

### DISCUSSION THREAD

**Discussion by asartalo** [vercel/community#859] [Help] [answered]
Hello. I wanted to move my project (https://brainchildprints.herokuapp.com/) to Vercel today but it seems that github orgs can only use teams and was too quick to create it to realize that it isn't free. Sorry.

I was trying to delete the team I created but that doesn't seem to work. The "Delete Team" dialog wants me to input the team name ("brainchild-projects") but the verify field underneath it has a `maxlength` of 14 which my team name's character length is beyond that. I tried to get around that by editing the input `maxlength` through the Firefox Developer tools but that didn't work.

The team name is "Brainchild Projects" and the org is https://github.com/brainchild-projects if that helps. I've tried "Brainchild Projects" and "brainchild-projects" but both of those did not work.

![Screenshot from 2022-09-15 07-40-30](https://user-images.githubusercontent.com/310708/190282073-13a0b62c-f724-4fb9-b4eb-e258a507be72.png)

**Comment by leerob** [comment, marked as answer]
Hey! It's asking you to type the literal string "Delete my team" - sorry for the confusion. I'll see if there's anything we can do better here to make this more clear.

**Reply by asartalo** [reply]
Doh! Thanks. Ugh.
